.github/workflows/dogfood-gitops.yml

```diff
@@ -48,13 +48,6 @@ jobs:
           ref: main
           path: fleet-gitops
 
-      - name: Apply env vars to profiles
-        env:
-          MANAGED_CHROME_ENROLLMENT_TOKEN: ${{ secrets.CLOUD_MANAGEMENT_ENROLLMENT_TOKEN }}
```

Comment:
Move this one variable definition to the step below.

Reply:
Done